src/commands/setAppWizardContextDefault.ts

```diff
@@ -0,0 +1,66 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as fse from 'fs-extra';
+import * as path from 'path';
+import { Uri, workspace, WorkspaceConfiguration } from 'vscode';
+import { IAppServiceWizardContext, javaUtils, LinuxRuntimes, WebsiteOS } from 'vscode-azureappservice';
+import { LocationListStep } from 'vscode-azureextensionui';
+import { configurationSettings, extensionPrefix } from '../constants';
+
+export async function setAppWizardContextDefault(wizardContext: IAppServiceWizardContext): Promise<void> {
+    const isJavaProject: boolean = await javaUtils.isJavaProject();
```

Comment:
`javaUtils` should also be moved from the `vscode-azureappservice` package to this repo

Reply:
Okay to do this is another PR?

https://github.com/Microsoft/vscode-azureappservice/issues/979

Reply:
Honestly I think I would block on this one. `javaUtils` was never supposed to be in `vscode-azureappservice` and we only allowed it in temporarily because we had a release coming up. This is definitely on-topic for the PR, it's should be an easy copy/paste of the file, and that way you can include it in the existing breaking-change release of the shared package instead of doing another breaking change release right after.